src/pykeen/regularizers.py

```diff
@@ -46,20 +46,17 @@ class Regularizer(nn.Module, ABC):
 
     def __init__(
         self,
-        device: torch.device,
         weight: float = 1.0,
         apply_only_once: bool = False,
     ):
         super().__init__()
-        self.device = device
-        self.register_buffer(name='weight', tensor=torch.as_tensor(weight, device=self.device))
+        self.register_buffer(name='weight', tensor=torch.as_tensor(weight))
         self.apply_only_once = apply_only_once
         self.updated = False
         self.reset()
 
     def to(self, *args, **kwargs) -> Regularizer:  # noqa: D102
```

Comment:
We do not need this method anymore - it was a dirty hack to manually move the regularization term along the module. Since we now use a buffer, pytorch knows what to do with it 🙂

Reply:
the to() method? okay lets nix it

Reply:
👍